indico/modules/events/contributions/models/fields.py

```diff
@@ -99,14 +99,13 @@ class ContributionField(db.Model):
     # - abstract_values (AbstractFieldValue.contribution_field)
     # - contribution_values (ContributionFieldValue.contribution_field)
 
-    @property
-    def field(self):
+    def get_field(self, management=False):
```

Comment:
Seems a bit cryptic to me. TBH, the current solution already seems pretty exaggerated to me. I cannot think of anything better, however.